public/index.html

```diff
@@ -0,0 +1,171 @@
+            <time class="commerce-dates" datetime="2023-11-17">2023/11/17</time>
+            <div class="commerce-tags tag1">お知らせ</div>
+            <p class="commerce-description">
+              ハウスメーカー・工務店向け無料ホームページテンプレート tp_housebuilder1公開。
+            </p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag2">施工事例</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag3">キャンペーン</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag4">お知らせ</div>
+            <p class="commerce-description">サンプルテキスト。サンプルテキスト。サンプルテキスト。</p>
+          </div>
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">20XX/00/00</time>
+            <div class="commerce-tags tag5">施工事例</div>
+            <p class="commerce-description">
+              サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。サンプルテキスト。
+            </p>
+          </div>
+        </div>
+        <div class="template-info">
+          <h1>テンプレートのご利用前に必ずお読み下さい</h1>
+          <hr />
+          <h3>利用規約のご案内</h3>
+          <p>
+            このテンプレートは、<span><a href="#">Template Party</a></span>
+            にて無料配布している『ハウスメーカー・工務店向け無料ホームページテンプレート
+            tp_housebuilder1』です。必ずダウンロード先のサイトの利用規約をご一読の上でご利用下さい。
+          </p>
+          <p></p>
+          <p class="warning-message">HP最下部の著作表示『Web Design:Template-Party』は無断で削除しないで下さい。</p>
+          <p>わざと見えなく加工する事も禁止です。 下部の著作を外したい場合は</p>
+          <p>Template-Partyのライセンス契約を行う事でHP下部の著作を外す事ができます。</p>
+          <h3>当テンプレートの詳しい使い方は</h3>
+
+          <div class="template-info-link"><a href="http://">こちらをご覧下さい。</a></div>
+        </div>
+        <div class="cms-info">
+          <h1>CMSのご案内</h1>
+          <hr />
+          <p>
+            2022年まで当サイトで販売窓口を行っていたCMSが、本来の開発者のサイトからの販売に切り替わりました。
+            「お知らせ」向けの無料CMSもあります。
+          </p>
+          <div class="template-info-banner-link"><a href="http://">&gt;&gt; クイックCMS様</a></div>
+          <img class="cms-info-img" src="/public/assets/banner_q19.jpg" alt="house banners" />
+        </div>
+      </section>
+    </main>
+    <nav>
+      <div class="footer-nav section-footer-nav">
+        <a href="http://">ホーム</a>
+        <a href="http://">会社概要</a>
+      </div>
+    </nav>
+    <footer class="footer section-footer container">
+      <h3>Copyright© SAMPLE HOME All Rights Reserved.</h3>
+      <h3>《Web Design:Template-Party》</h3>
```

Comment:
[IMO]
This sentence is not a heading for this website. Using paragraph or [small](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small) tag is better. 
As I were a coder for this website, I'd choose `small`.

Reply:
@sharray2918 
Thanks for letting me know about this section!
I just learnt something new! I will make sure to use `small` going forth!